### ISSUE TITLE
Bad IS and SSIM score

### ISSUE THREAD

**Issue by dawn7564** [tengteng95/Pose-Transfer#51]
Really really thanks for your awesome works. 

However, using your code and hyper-parameter, I got bad evaluation results after training by myself:
SSIM=0.268 IS=3.652 mask-SSIM=0.792 mask-IS=3.733

Meanwhile, using your provided pre-trained model, I got almost right results:
SSIM=0.311 IS=3.329 mask-SSIM=0.811 mask-IS=3.778

My training environment is pytorch0.3.1,  I don't know why the results are different. 

Thanks again~

**Comment by tengteng95** [comment]
Hi, it is really strange to obtain such an unsatisfying result.  I could not tell where is wrong at the moment since too few information is provided. Do you make any modifications to the original code?  Does the training process converge well?

**Comment by dawn7564** [comment]
Hi, thanks for replying : ) 
I didn't modify any code, I just download the vgg19-dcbb9e9d.pth by myself and put it on the .torch/models/ because it's slow downloading by program.

and training process is here
![pytorch0-3-1train0-3-1](https://user-images.githubusercontent.com/45144565/69792125-eb960980-1200-11ea-99e7-61f2a014f7e0.png)

My training and testing environment(conda):
pytorch0.3.1 cuda90; torchvision0.2.0; python3.6.9;

By the way, I also use pytorch1.3 to run codes pytorch_v1.0 and get almost the same bad results...

**Comment by chengaopro** [comment]
> Really really thanks for your awesome works.
> 
> However, using your code and hyper-parameter, I got bad evaluation results after training by myself:
> SSIM=0.268 IS=3.652 mask-SSIM=0.792 mask-IS=3.733
> 
> Meanwhile, using your provided pre-trained model, I got almost right results:
> SSIM=0.311 IS=3.329 mask-SSIM=0.811 mask-IS=3.778
> 
> My training environment is pytorch0.3.1, I don't know why the results are different.
> 
> Thanks again~



> Really really thanks for your awesome works.
> 
> However, using your code and hyper-parameter, I got bad evaluation results after training by myself:
> SSIM=0.268 IS=3.652 mask-SSIM=0.792 mask-IS=3.733
> 
> Meanwhile, using your provided pre-trained model, I got almost right results:
> SSIM=0.311 IS=3.329 mask-SSIM=0.811 mask-IS=3.778
> 
> My training environment is pytorch0.3.1, I don't know why the results are different.
> 
> Thanks again~

Hi, I strictly followed their instruction and environment, use their provided pre-trained model, but I only got IS=3.159, mask-IS=3.735, SSIM=0.2815, mask-SSIM=0.799 on Market-1501 dataset, which is worse than those in original paper. I saw you reproduced their performance, so what do you think is the cause of this gap?
By the way, I have reproduced their performance on DeepFashion dataset. But the issue in Market-1501 dataset has troubled me for a long time... , thanks

**Comment by chengaopro** [comment]
> Really really thanks for your awesome works.
> 
> However, using your code and hyper-parameter, I got bad evaluation results after training by myself:
> SSIM=0.268 IS=3.652 mask-SSIM=0.792 mask-IS=3.733
> 
> Meanwhile, using your provided pre-trained model, I got almost right results:
> SSIM=0.311 IS=3.329 mask-SSIM=0.811 mask-IS=3.778
> 
> My training environment is pytorch0.3.1, I don't know why the results are different.
> 
> Thanks again~

Hi, I strictly followed their instruction and environment, use their provided pre-trained model, but I only got IS=3.159, mask-IS=3.735, SSIM=0.2815, mask-SSIM=0.799 on Market-1501 dataset, which is worse than those in original paper. I saw you reproduced their performance, so what do you think is the cause of this gap? @dawn7564 @tengteng95 
By the way, I have reproduced their performance on DeepFashion dataset. But the issue in Market-1501 dataset has troubled me for a long time... , thanks

**Comment by dawn7564** [comment]
Hi, i have got the almost right results now by redownloading the dataset, files, and building new conda environment. Frankly, i dont know why the results come right after that. Maybe you can try my way.

I use pytorch v1.0.1, python3.6 for training and testing, if it helps.
